plugins/lime-plugin-fbw/src/containers/Scan.js

```diff
@@ -148,20 +172,14 @@ export const Scan = ({ searchNetworks, setNetwork, toggleForm, status, networks
 				</div>
 			</div>
 			{state.error && <Toast text={<Trans>Must select a network and a valid hostname</Trans>} />}
+			{/* todo(kon) create scanning error toast */}
 			{(status === 'scanning' && <Toast text={<Trans>Scanning for existing networks</Trans>} />)}
 		</div>
 	);
 };
 
-const mapStateToProps = (state) => ({
-	networks: state.firstbootwizard.networks,
-	status: state.firstbootwizard.status
-});
-
-const mapDispatchToProps = (dispatch) => ({
-	searchNetworks: bindActionCreators(searchNetworks ,dispatch),
-	setNetwork: bindActionCreators(setNetwork ,dispatch),
-	showNotification: bindActionCreators(showNotification, dispatch)
-});
-
-export default connect(mapStateToProps, mapDispatchToProps)(Scan);
+const getApName = ({ ap = '', file = '' }) => {
```

Comment:
It's okey to do this hear, but just to give you options:
This data enrichment could also be done at query level.
You can check the "useAssocList" query to checkout what I mean.